examples/web/child.html

```diff
@@ -1,29 +0,0 @@
-﻿<!DOCTYPE html>
```

Comment:
This is no longer needed as we simply just recursively show index.html in the children windows